.github/workflows/build-images.yaml

```diff
@@ -208,10 +209,9 @@ jobs:
           image: ${{ env.OPERATOR_NAME }}-catalog
           tags: ${{ env.IMG_TAGS }}
           platforms: linux/amd64,linux/arm64
```

Comment:
not related to this PR, but I do not think that catalog image needs to be multi-platform. It's just a single yaml file!

Reply:
Could pulling the image from a different platform fail if only amd64 is available (despite being just a single yaml inside)?

Reply:
The image is being pulled by OLM. So, not being entirely sure, this needs to be checked first :+1:

Reply:
From [OS docs](https://docs.openshift.com/container-platform/4.5/operators/admin/olm-managing-custom-catalogs.html):

> Set --filter-by-os to the operating system and architecture to use for the base image, which must match the target OpenShift Container Platform cluster. Valid values are linux/amd64, linux/ppc64le, and linux/s390x.

Which not sure if it could be interpreted to also the actual catalog yaml img that it's delivered